dio/lib/src/multipart_file.dart

```diff
@@ -7,6 +7,8 @@ import 'multipart_file/io_multipart_file.dart'
     if (dart.library.html) 'multipart_file/browser_multipart_file.dart';
 import 'utils.dart';
 
+typedef DioMediaType = MediaType;
```

Comment:
Also, we don't want to reduce our docs coverage during changes, so maybe:
```suggestion
/// The type (alias) for specifying the content-type of the `MultipartFile`. 
typedef DioMediaType = MediaType;
```